dev-libs/xslt.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import stdlib
+from stdlib.manifest import manifest
+from stdlib.template import autotools
+from stdlib.template.configure import configure
+from stdlib.template.make import make
+from stdlib.split.drain_all import drain_all
+
+def xslt_configure():
+    stdlib.cmd('sed -i s/3000/5000/ libxslt/transform.c doc/xsltproc.{1,xml}')
```

Comment:
Would you mind commenting on this change? Why is it needed, what does it provide?

Reply:
It increases the recursion limit in libxslt which is needed by some packages for their documentation. (according to [LFS](http://www.linuxfromscratch.org/blfs/view/svn/general/libxslt.html))
I figured it might be useful

Reply:
I meant in the manifest :p

Reply:
Without it, that recursion limit won't be increased anymore so when someone installs this package, the documentation of packages requiring it will be broken. So it's better to keep it in the manifest, I think? I'm not too sure on what you mean.

Reply:
Comment on the change within the manifest. As a python comment, to explain why x)

Reply:
ooooh ok I get it now